<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ashish-PC" name="Default test" tests="2" failures="2" timestamp="13 Mar 2015 02:51:45 GMT" time="3.728" errors="0">
  <testcase name="modifyPortFolioStockTest" time="3.056" classname="com.qtpselenium.framework.suiteB.ModifyPortFolioStockTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.qtpselenium.framework.util.Keywords.takeScreenShot(Keywords.java:630)
	at com.qtpselenium.framework.util.Keywords.reportError(Keywords.java:613)
	at com.qtpselenium.framework.util.Keywords.openBrowser(Keywords.java:213)
	at com.qtpselenium.framework.util.Keywords.executeKeywords(Keywords.java:84)
	at com.qtpselenium.framework.suiteB.ModifyPortFolioStockTest.modifyPortFolioStockTest(ModifyPortFolioStockTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- modifyPortFolioStockTest -->
  <testcase name="modifyPortFolioStockTest" time="3.056" classname="com.qtpselenium.framework.suiteB.ModifyPortFolioStockTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.qtpselenium.framework.util.Keywords.takeScreenShot(Keywords.java:630)
	at com.qtpselenium.framework.util.Keywords.reportError(Keywords.java:613)
	at com.qtpselenium.framework.util.Keywords.openBrowser(Keywords.java:213)
	at com.qtpselenium.framework.util.Keywords.executeKeywords(Keywords.java:84)
	at com.qtpselenium.framework.suiteB.ModifyPortFolioStockTest.modifyPortFolioStockTest(ModifyPortFolioStockTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- modifyPortFolioStockTest -->
</testsuite> <!-- Default test -->
